app.py

```diff
@@ -37,13 +56,33 @@ def store_duty_view():
     duties = cursor.fetchall()
     return render_template('store/duty/view.j2', duties=duties)
 
-@app.route('/store/duty/add')
+@app.route('/store/duty/add', methods = ['GET', 'POST'])
 def store_duty_add():
+    if request.method == 'POST':
+        name = request.form['name']
+        data = (name)
+        query = "INSERT INTO Duties (name) VALUES ('%s');" % data
+        cursor = db.execute_query(db_connection=db_connection, query=query)
+
     return render_template('store/duty/add.j2')
 
-@app.route('/store/duty/assign')
+@app.route('/store/duty/assign', methods = ['GET', 'POST'])
 def store_duty_assign():
-    return render_template('store/duty/assign.j2')
+    if request.method == 'POST':
+        employee_id = request.form['employee']
+        duty_id = request.form['duty']
+        data = (employee_id, duty_id)
+        query = "INSERT IGNORE INTO EmployeeDuties (employeeId, dutyId) VALUES ('%s', '%s');" % data
```

Comment:
'IGNORE' suppresses an error that would occur if the user attempts to assign an employee to the same duty more than once.  Duplicate rows are NOT added.